test/terra/backends/aer_simulator/test_standard_gates.py

```diff
@@ -84,8 +84,8 @@
     (C3XGate, 0, False),
     (C4XGate, 0, False),
     # Parameterized Gates
-    (RC3XGate, 1, False),
-    (RCCXGate, 1, False),
+    (RC3XGate, 0, False),
+    (RCCXGate, 0, False),
```

Comment:
What's this change about?

Reply:
The second argument means a number of parameters. This was a bug in previous code because [RC3XGate](https://github.com/Qiskit/qiskit-terra/blob/main/qiskit/circuit/library/standard_gates/x.py#L685) and [RCCXGate](https://github.com/Qiskit/qiskit-terra/blob/f134aeb7a9aa6f7381de5a9c9de0fb662fea3c5f/qiskit/circuit/library/standard_gates/x.py#L404) are 3-controlled Toffoli gate and normal Toffoli gate in Terra. In the previous code, a float number was specified as `label` but this becomes a fail since [recent change](https://github.com/Qiskit/qiskit-terra/commit/ef997f9ba2c3da0b1018394cd1a2649bdeb6a183) in Terra.